packages/database/prisma/schema.prisma

```diff
@@ -196,6 +197,39 @@ model Invite {
   @@map("invite")
 }
 
+model Draft {
+  id              String   @id @default(cuid())
+  teamId          String   @unique
+  hackathonId     String   
+  name            String?  
+  tagline         String?  
+  description     String?  
+  githubUrl       String?  
+  videoUrl        String?  
+  images          String[] @default([]) 
+  createdAt       DateTime @default(now())
+  updatedAt       DateTime @updatedAt
+
+  // Relations
+  team         Team        @relation(fields: [teamId], references: [id], onDelete: Cascade)
+  draftTracks  DraftTrack[]
+
+  @@map("draft")
+}
+
+// Join table: Draft ↔ Track (drafts can be in multiple tracks)
+model DraftTrack {
+  draftId String
+  trackId String
+
+  // Relations
+  draft   Draft @relation(fields: [draftId], references: [id], onDelete: Cascade)
+  track   Track @relation(fields: [trackId], references: [id], onDelete: Cascade)
+
+  @@id([draftId, trackId])
+  @@map("draft_track")
+}
```

Comment:
why not use an implicit join if this table doesn't have any additional fields? as in, just make draft have an array of tracks